ui/ozone/platform/wayland/wayland_window.cc

```diff
@@ -101,11 +161,19 @@ void WaylandWindow::SetTitle(const base::string16& title) {
 }
 
 void WaylandWindow::SetCapture() {
-  NOTIMPLEMENTED();
+  if (g_current_capture_ == this)
+    return;
+
+  WaylandWindow* old_capture = g_current_capture_;
+  if (old_capture)
+    old_capture->delegate()->OnLostCapture();
+
+  g_current_capture_ = this;
 }
 
 void WaylandWindow::ReleaseCapture() {
-  NOTIMPLEMENTED();
+  if (g_current_capture_ == this)
+    g_current_capture_ = nullptr;
```

Comment:
Actually, if we have a menu window open and click on a main window (the one in the background), OnLostCapture tells menu_host that we don't have capture any more and that one closes the menu and its submenus